nexus/users/services.py

```diff
@@ -13,10 +13,9 @@ def get_and_authenticate_user(email, password):
     return user
 
 
-def create_user_account(email, password, first_name="", last_name=""):
+def create_user_account(email, password, first_name="", last_name="", **extra_fields):
     user = get_user_model().objects.create_user(
-        email=email, password=password, first_name=first_name, last_name=last_name
-    )
+        email=email, password=password, first_name=first_name, last_name=last_name, **extra_fields)
```

Comment:
Next time, rather than changing the closing brackets and stuff, you can just put the extra content on new line :)